### PR TITLE
Add howto add a dependency to README.md

### PR DESCRIPTION
**Pull request by tisto**

### CODE REVIEW

**Comment by tisto** [comment]
@ericof not sure where to put this. Let me know where docs like this would belong and I am happy to amend my PR.

File: README.md [hunk 1]
```diff
@@ -159,6 +159,47 @@ The configuration for a new Volto site is:
 }
 ```
 
+#### How to add an add-on
+
+If you want to add a Plone backend add-on to your Plone distribution you have to:
```
Comment (stevepiercy) [inline README.md:164]:
```suggestion
If you want to add a Plone backend add-on to your Plone distribution, then you must perform the following steps.
```

File: README.md [hunk 2]
```diff
@@ -159,6 +159,47 @@ The configuration for a new Volto site is:
...
+
+Add your add on (e.g. collective.person) to your setup.py:
```
Comment (stevepiercy) [inline README.md:166]:
```suggestion
Add your add-on, such as `collective.person`, to your `setup.py`:
```

File: README.md [hunk 3]
```diff
@@ -159,6 +159,47 @@ The configuration for a new Volto site is:
...
+
+```
```
Comment (stevepiercy) [inline README.md:168]:
````suggestion
```python
````

File: README.md [hunk 4]
```diff
@@ -159,6 +159,47 @@ The configuration for a new Volto site is:
...
+    install_requires=[
+        "setuptools",
+        "Plone",
+        "plone.distribution>=1.0.0b2",
+        "plone.api",
+        "collective.person",
+    ],
+```
+
+Add it to your dependencies.zcml:
```
Comment (stevepiercy) [inline README.md:178]:
```suggestion
Add it to your `dependencies.zcml`:
```

File: README.md [hunk 5]
```diff
@@ -159,6 +159,47 @@ The configuration for a new Volto site is:
...
+
+```
```
Comment (stevepiercy) [inline README.md:180]:
````suggestion
```xml
````

File: README.md [hunk 6]
```diff
@@ -159,6 +159,47 @@ The configuration for a new Volto site is:
...
+  <!-- List all packages you depend here -->
+  <include package="plone.volto" />
+  <include package="plone.restapi" />
+  <include package="collective.person" />
+  <include package="plone.distribution" />
+
+</configure>
+```
+
+Add it to your profiles.json:
```
Comment (stevepiercy) [inline README.md:190]:
```suggestion
Add it to your `profiles.json`:
```

File: README.md [hunk 7]
```diff
@@ -159,6 +159,47 @@ The configuration for a new Volto site is:
...
+
+```
```
Comment (stevepiercy) [inline README.md:192]:
````suggestion
```json
````

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Added some GitHub flavor Markdown markup.

**Comment by stevepiercy** [comment]
@tisto I'd suggest a new section in https://github.com/plone/plone.distribution/blob/main/docs/usage/code-examples.md

**Review by stevepiercy — APPROVED** [review]
LGTM, unless you want to move it into another location in the docs.

**Pull request merged**